packages/discord.js/src/managers/GuildMemberManager.js

```diff
@@ -336,11 +341,11 @@ class GuildMemberManager extends CachedManager {
       query.include_roles = dry ? resolvedRoles.join(',') : resolvedRoles;
```

Comment:
Ditto, this might be set to undefined if missing

Reply:
Can't be missing at this point